app/src/sharedTest/java/org/oppia/android/app/topic/practice/TopicPracticeFragmentTest.kt

```diff
@@ -379,14 +383,34 @@ class TopicPracticeFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicPracticeFragment_createFragmentWithProfileId_verifyProfileIdInBundle() {
+    launchTopicActivityIntent(profileId, FRACTIONS_TOPIC_ID).use {
+      clickPracticeTab()
+      clickPracticeItem(position = 1, targetViewId = R.id.subtopic_check_box)
+      onView(isRoot()).perform(orientationLandscape())
+      scrollToPosition(position = 5)
+
+      it.onActivity {
+        activity ->
```

Comment:
```suggestion
      it.onActivity { activity ->
```

Wrapping is off here. Ditto everywhere else in the PR.

Reply:
Updated.